### ISSUE TITLE
Bug: false-positive when unary operators are used with $object->property

### ISSUE THREAD

**Issue by istrof** [djoos/Symfony-coding-standard#129]
False-positive for:
```php
$foo->bar++
```
Because bar is parsed as a T_STRING and not T_VARIABLE
Additional check for T_OBJECT_OPERATOR (->) covers this case: https://github.com/djoos/Symfony-coding-standard/pull/128
